apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableToContainInAnyOrderAtLeast1EntriesExpectationsBuilderTest.kt

```diff
@@ -0,0 +1,51 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.integration.AbstractIterableToContainInAnyOrderAtLeast1EntriesExpectationsTest
+import kotlin.test.Test
+
+class IterableToContainInAnyOrderAtLeast1EntriesExpectationsBuilderTest :
+    AbstractIterableToContainInAnyOrderAtLeast1EntriesExpectationsTest(
+        functionDescription to Companion::toContainInAnyOrderEntries,
+        functionDescription to Companion::toContainNullableEntries
+    ) {
+
+    companion object : IterableToContainSpecBase() {
+        val functionDescription = "$toContain.$inAnyOrder.$atLeast(1).$entry/$entries"
+
+
+        private fun toContainInAnyOrderEntries(
+            expect: Expect<Iterable<Double>>,
+            a: Expect<Double>.() -> Unit,
+            aX: Array<out Expect<Double>.() -> Unit>
+        ): Expect<Iterable<Double>> =
+            if (aX.isEmpty()) expect.toContain.inAnyOrder.atLeast(1).entry(a)
+            else expect.toContain.inAnyOrder.atLeast(1).entries(a, *aX)
+
+        private fun toContainNullableEntries(
+            expect: Expect<Iterable<Double?>>,
+            a: (Expect<Double>.() -> Unit)?,
+            aX: Array<out (Expect<Double>.() -> Unit)?>
+        ): Expect<Iterable<Double?>> =
+            if (aX.isEmpty()) expect.toContain.inAnyOrder.atLeast(1).entry(a)
+            else expect.toContain.inAnyOrder.atLeast(1).entries(a, *aX)
+    }
+
+    @Suppress("AssignedValueIsNeverRead", "UNUSED_VALUE")
+    @Test
+    fun ambiguityTest() {
+        var list: Expect<List<Number>> = expect(listOf(1))
+        var nList: Expect<Set<Number?>> = expect(setOf(1, null))
```

Comment:
```suggestion
        var nSet: Expect<Set<Number?>> = expect(setOf(1, null))
```
(was wrong before, if you like you could improve it)

Reply:
@robstoll - Done